inc/class-wpseo-meta.php

```diff
@@ -1046,7 +1046,7 @@ public static function keyword_usage( $keyword, $post_id ) {
 		if ( WPSEO_Utils::is_yoast_seo_premium() ) {
 			$query['meta_query'][] = array(
 				'key'     => '_yoast_wpseo_focuskeywords',
-				'value'   => sprintf( '"keyword":"%s"', $keyword ),
+				'value'   => sprintf( '"keyphrase":"%s"', $keyword ),
```

Comment:
This case isn't translatable?

Reply:
Ouch this is actually part of the query. I guess it should be `keyword`.